### ISSUE TITLE
powerpc64: Warning: invalid register expression

### ISSUE THREAD

**Issue by monniaux** [AbsInt/CompCert#343]
When compiling the standard library on ppc64:
```
powerpc64/i64_utof.s: Assembler messages:
powerpc64/i64_utof.s:52: Warning: invalid register expression
```

I don't know the ppc platform so I don't know what's right or not.

**Comment by bschommer** [comment]
PowerPC64 is currently not supported.

**Comment by xavierleroy** [comment]
This could actually be a typo in the assembly code.  I'm looking into it.

**Comment by xavierleroy** [comment]
This warning was a smoking gun leading to a actual, if hard to trigger, error.  This is fixed in f1abe04e .
